mapie/tests/test_utils.py

```diff
@@ -410,3 +410,22 @@ def test_change_values_zero_one() -> None:
     """Test that binary output are changed to zero one outputs."""
     array_ = check_binary_zero_one(np.array([0, 4, 4]))
     assert (np.unique(array_) == np.array([0, 1])).all()
+
+
+# Valid Gamma Test
+def test_valid_gamma():
+    valid_gamma = 0.5
+    assert check_gamma(valid_gamma) is None
```

Comment:
STYLE: add it in the description of the function
```suggestion
@pytest.mark.parametrize("gamma", [0.1, 0.5, 0.9])
def test_valid_gamma(gamma: float) -> None:
    """Test a valid gamma parameter."""
    assert check_gamma(gamma) is None
```